### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by loannaflip**

### CODE REVIEW

**Comment by loannaflip** [comment]
ok. take

File: README.md [hunk 1]
```diff
@@ -12,8 +12,19 @@ Novel coronavirus (2019-nCoV) is coronavirus originated in Wuhan, China. [[1]](h
...
 ## Symptoms
-FIXME: Fever - verification needed
-FIXME: Respiratory issues - verification needed
+* Fever
+* Cough
+* Shortness of breath
+* Runny nose
+* Headache
+* Sore throat
+* A general feeling of being unwell
```
Comment (Kreyren) [inline README.md]: reference required

File: README.md [hunk 2]
```diff
@@ -12,8 +12,19 @@ Novel coronavirus (2019-nCoV) is coronavirus originated in Wuhan, China. [[1]](h
...
+
+## Avoiding 
+<a href=https://www.who.int/emergencies/diseases/novel-coronavirus-2019/advice-for-public>Advice</a> For Public
+
+## Note
+In January 2020, several organizations and institutions began work on creating vaccines for 2019 n-CoV based on the published genome.
```
Comment (Kreyren) [inline README.md]: reference required

File: README.md [hunk 3]
```diff
@@ -12,8 +12,19 @@ Novel coronavirus (2019-nCoV) is coronavirus originated in Wuhan, China. [[1]](h
...
+* Shortness of breath
+* Runny nose
+* Headache
+* Sore throat
+* A general feeling of being unwell
+
+## Avoiding 
+<a href=https://www.who.int/emergencies/diseases/novel-coronavirus-2019/advice-for-public>Advice</a> For Public
+
+## Note
```
Comment (Kreyren) [inline README.md]: place this in a header

File: README.md [hunk 4]
```diff
@@ -12,8 +12,19 @@ Novel coronavirus (2019-nCoV) is coronavirus originated in Wuhan, China. [[1]](h
...
-FIXME: Respiratory issues - verification needed
+* Fever
+* Cough
+* Shortness of breath
+* Runny nose
+* Headache
+* Sore throat
+* A general feeling of being unwell
+
+## Avoiding
```
Comment (Kreyren) [inline README.md]: Place this in the header

File: README.md [hunk 5]
```diff
@@ -12,8 +12,19 @@ Novel coronavirus (2019-nCoV) is coronavirus originated in Wuhan, China. [[1]](h
 FIXME: Man are more susceptible to 2019-nCoV - Verificatio needed
 
 ## Symptoms
-FIXME: Fever - verification needed
-FIXME: Respiratory issues - verification needed
+* Fever
+* Cough
+* Shortness of breath
+* Runny nose
```
Comment (Kreyren) [inline README.md]: I think you meant `rhinitis` here

File: README.md [hunk 6]
```diff
@@ -12,8 +12,19 @@ Novel coronavirus (2019-nCoV) is coronavirus originated in Wuhan, China. [[1]](h
 FIXME: Man are more susceptible to 2019-nCoV - Verificatio needed
 
 ## Symptoms
-FIXME: Fever - verification needed
-FIXME: Respiratory issues - verification needed
+* Fever
+* Cough
+* Shortness of breath
```
Comment (Kreyren) [inline README.md]: Did you mean `dyspnea`?

**Pull request closed without merging**